addons/io/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -498,7 +498,7 @@ public void sendItemUpdate(String itemName, String itemState) {
                 itemUpdateMessage.put("itemStatus", itemState);
                 socket.emit("itemupdate", itemUpdateMessage);
             } catch (JSONException e) {
-                logger.error(e.getMessage());
+                logger.error("{}", e.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace